### ISSUE TITLE
[TW-1432] task start/stop can be issued on completed tasks

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1459]
_Renato Alves on 2014-10-14T14:34:13Z says:_ 

 As of taskwarrior 2.4.0 one can start/stop a completed (done) task without affecting it's state.

I think one of the two behaviours should be adopted:
1.  start/stop is not allowed on complete tasks
1.  stop shouldn't be allowed; start could be allowed but it would implicitly change the task state to pending.

Since "done" already implicitly stops the timer (TW-252), using stop with complete tasks makes no sense.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-10-14T14:34:13Z
Modified: 2015-05-28T08:18:37Z
```

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2014-10-14T20:18:41Z says:_ 

 I'd adjust #2 to this:
Starting a completed task should set its status to pending, but there is no need to disallow stopping a completed task. That would only lead to problems for people trying to clean up after tasks that were marked completed while active in a previous Taskwarrior version, and after TW-252 is fixed there won't be any new tasks with this "faulty" status.

Note: I am not saying #2 is the way to go, just that this would be a better #2

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2015-05-28T08:18:37Z says:_ 

 Fixed by going with "#2" as described.
